packages/@dcl/react-ecs/src/components/uiTransform/index.ts

```diff
@@ -69,28 +73,34 @@ const defaultUiTransform: PBUiTransform = {
 /*#__PURE__*/
 export function parseUiTransform(props: UiTransformProps = {}): PBUiTransform {
   const {
-    position,
-    padding,
-    margin,
     height,
     minHeight,
     maxHeight,
     width,
-    maxWidth,
     minWidth,
+    maxWidth,
     ...otherProps
   } = props
   return {
     ...defaultUiTransform,
     ...otherProps,
-    ...parsePosition(position, 'position'),
-    ...parsePosition(margin, 'margin'),
-    ...parsePosition(padding, 'padding'),
-    ...parseSize(height, 'height'),
-    ...parseSize(minHeight, 'minHeight'),
-    ...parseSize(maxHeight, 'maxHeight'),
-    ...parseSize(width, 'width'),
-    ...parseSize(minWidth, 'minWidth'),
-    ...parseSize(maxWidth, 'maxWidth')
+    ...parsePosition(props.position, 'position'),
+    ...parsePosition(props.margin, 'margin'),
+    ...parsePosition(props.padding, 'padding'),
+    ...parseSize(props.height, 'height'),
+    ...parseSize(props.minHeight, 'minHeight'),
+    ...parseSize(props.maxHeight, 'maxHeight'),
+    ...parseSize(props.width, 'width'),
+    ...parseSize(props.minWidth, 'minWidth'),
+    ...parseSize(props.maxWidth, 'maxWidth'),
+    ...getDisplay(props.display),
+    ...getAlign('alignContent', props.alignContent, YGAlign.YGA_FLEX_START),
+    ...getAlign('alignSelf', props.alignSelf, YGAlign.YGA_FLEX_START),
+    ...getAlign('alignItems', props.alignItems, YGAlign.YGA_FLEX_START),
+    ...getJustify(props.justifyContent),
+    ...getFlexDirection(props.flexDirection),
+    ...getFlexWrap(props.flexWrap),
+    ...getOverflow(props.overflow),
+    ...getPoistionType(props.positionType)
```

Comment:
just thinking about this (probably not for this PR)...

What about an approach where we have a dictionary of css props mapping to their corresponding parsing fn (ex: `{ position: parsePosition, height: parseSize, etc }`) that allows us to create a an executor that basically runs against all the keys in the `props` field and run the proper parser? 
That way, adding a new parser is just creating the function and map it to the dictionary, like "installing" it, and we avoid having to remember to update all this stuff

Reply:
i didn't get this one 🤔

Reply:
something like this? => https://tsplay.dev/mAdRPN

Reply:
could be.
The only problem that i see is when the value is not defined, if we create the prop then protobuf will serialize that value. So we need to do something like filter undefined values, etc.
It's more readable that way, but its the same that the interface is doing right now. It's too much work for something more readable, maybe in another iteration.